python/cuspatial/cuspatial/_lib/spatial.pyx

```diff
@@ -13,6 +13,11 @@ from cudf._lib.legacy.cudf import *
 from cuspatial._lib.cpp.coordinate_transform cimport (
```

Comment:
I didn't really understand why everything was lumped in a file called "spatial", which is why I created a new file for spatial_window...

Perhaps distance metrics should get their own file too? (Distance_metrics.pyx)?

Reply:
The python community as a whole seems to like keeping everything in the same file. :disappointed: 

https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/cudf/core/dataframe.py

@thomcom any opinion here

Reply:
Yeah I agree. Honestly I'd be fine with naming it after the C++ header like we do in cudf. Also I just noticed we need to fix the Cython build's include paths. The header paths should be this:

```diff
- cdef extern from "haversine.hpp" namespace "cuspatial" nogil:
+ cdef extern from "cuspatial/haversine.hpp" namespace "cuspatial" nogil:
```